src/crons/retry-error.rb

```diff
@@ -0,0 +1,89 @@
+#!/bin/ruby
+require 'yaml'
+require 'fileutils'
+require 'net/smtp'
+
+#  retry-error
+# ==================================================
+# Name : David J. Davis
+# Date :  4/11/2017
+#
+# Requirements:  You must run add_project_env.sh first
+# so that the environmental variable is able to be used.
+#
+# Useage: From the terminal as root user run the command
+# ruby ./retry-error.rb {filename.yaml}
```

Comment:
I debated on that for a while.  Should the extension be required in the commandline argument or should it just be hard coded?  I was thinking that the extension had to be in the argument so that if it did change the code wouldn't have to be changed, but looking at the code base it depends on yaml so either way the code base will have to be changed.